easybuild/easyconfigs/a/AmrPlusPlus/AmrPlusPlus-2.0-GCC-8.3.0.eb

```diff
@@ -0,0 +1,74 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage: 		https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::		Robert Qiao <rob.qiao@flinders.edu.au>
+# Software License::	MIT
+#
+# Notes::
+##
+
+easyblock = 'CmdCp'
+
+name = 'AmrPlusPlus'
+version = '2.0'
+
+github_account = 'meglab-metagenomics'
+
+homepage = 'https://megares.meglab.org/amrplusplus/latest/html/v2'
+description = """AmrPlusPlus v2.0 can process the raw data from the sequencer,
+identify the fragments of DNA, and count them. It also provides a count of the
+polymorphisms that occur in each DNA fragment with respect to the reference database.
+"""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+source_urls = ['https://github.com/%(github_account)s/%(name)s_v%(version_major)s/archive']
+sources = ['8afc0af5234f8e322cc0f5e40cc553ecd5c932b8.zip']
```

Comment:
Please instruct EasyBuild to rename after download, so you can easily tell which downloaded file corresponds to which version:

```python
sources = [{
    'download_filename': '%s.zip' % local_commit,
    'filename': SOURCE_ZIP,
}]
```

(`SOURCE_ZIP` is equivalent to `'%(name)s-%(version)s.zip'`)